packaging/aleph-vm/DEBIAN/control

```diff
@@ -3,6 +3,6 @@ Version: 0.1.8
 Architecture: all
 Maintainer: Aleph.im
 Description: Aleph.im VM execution engine
-Depends: python3,python3-pip,python3-aiohttp,python3-msgpack,python3-aiodns,python3-setproctitle,redis,python3-aioredis,python3-psutil,sudo,acl,curl,systemd-container,squashfs-tools,debootstrap
+Depends: python3,python3-pip,python3-aiohttp,python3-msgpack,python3-aiodns,python3-sqlalchemy,python3-setproctitle,redis,python3-aioredis,python3-psutil,sudo,acl,curl,systemd-container,squashfs-tools,debootstrap
```

Comment:
It would be better if you could fix the version of packages in a requirements.txt file or something. SQLAlchemy is moving towards v2.0 at the moment, there are already massive differences between 1.3 and 1.4 (the current version) so I'm pretty confident that we will have compatibility issues at some point.